crypto/fipsmodule/bcm.c

```diff
@@ -251,6 +251,10 @@ static void BORINGSSL_maybe_set_module_text_permissions(int _permission) {}
 
 #endif  // !ASAN
 
+#if defined(AWSLC_FIPS_FAILURE_CALLBACK)
```

Comment:
Does the reallyyyy need a compile-time flag?

Reply:
With this iteration of this design I think so. My fear is an application using AWS-LC for FIPS might not notice if something else in their dependencies defined the callback and then let the application continue using AWS-LC in a degraded state silently. And now this version doesn't abort so it's possible to continue using AWS-LC in a degraded state.